### PR TITLE
Feature global animations

### PR DESCRIPTION
**Pull request by jhsware**
- added animation of logo when navigating between pages
- updated documentation of inferno-animation

also includes all the features of https://github.com/infernojs/inferno-website/pull/158

### CODE REVIEW

**Comment by Havunen** [comment]
I was testing the website using my android phone and the main page seems broken :/

Can you check that please

File: .babelrc [hunk 1]
```diff
@@ -5,8 +5,7 @@
       {
         "targets": {
           "browsers": [ ">0.25%", "ie >= 11"]
-        },
-        "loose": true
+        }
```
Comment (Havunen) [inline .babelrc:8]: Is there some reason to remove "loose" statements, it used to compile it less strict es5 equilevant code which runs faster

Comment (jhsware) [reply .babelrc:8]: We were getting getting lots of warnings when transpiling, but perhaps it was premature if we don't also allow native class defs in compiled code. Perhaps worth moving the needle forward though, IE11 is the only reason to not generate class defs AFAIK.

**Comment by jhsware** [comment]
> I was testing the website using my android phone and the main page seems broken :/
> 
> 
> 
> Can you check that please

I can reproduce and am working on a solution.

**Comment by jhsware** [comment]
I believe I have fixed the issues on mobile devices.

File: .babelrc (hunk 1 above)
Comment (jhsware) [reply .babelrc]: So  I have silenced the warnings by adding loose: true to plugins instead. But perhaps we should start generating proper class defs or is it too soon? IE11 isn't something we want to support any more IMHO. fae6a6f

**Comment by jhsware** [comment]
@Havunen I have now tested on:
- macOS Chrome
- macOS Firefox
- macOS Safari
- iOS Mobile Safari

Switching between Home and API sometimes causes global animation to drop single frames on Chrome (barely noticeable) and appears to be due to content on API page. I haven't investigated further.

Not tested: Windows, Linux, Android, Edge

**Comment by jhsware** [comment]
This PR FIXES https://github.com/infernojs/inferno-website/issues/152

**Comment by Havunen** [comment]
Okay, lets merge this then. Its good promotion for the upcoming version so why not :)

**Pull request merged**